lib/src/components/NxSearchDropdown/NxSearchDropdown.tsx

```diff
@@ -89,6 +89,16 @@ function NxSearchDropdownRender<T extends string | number = string>(
     onSearch(value.trim());
   }
 
+  const handleKeyDown: KeyboardEventHandler = event => {
+    if (event.key === 'Escape') {
```

Comment:
I _think_ this is fine nowadays but I recall in the past different browsers had different ways to check for escape, so be sure this works in everything we currently support.

Reply:
> I _think_ this is fine nowadays but I recall in the past different browsers had different ways to check for escape, so be sure this works in everything we currently support.

Good point. I did look at NxDropdown implementation and it also only checks for `key === 'Escape'`. Do you think that's a safe assumption to say that it works with what we currently supports 🤔 ?
https://caniuse.com/keyboardevent-key

Reply:
> Good point. I did look at NxDropdown implementation and it also only checks for key === 'Escape'. Do you think that's a safe assumption to say that it works with what we currently supports thinking ?

Yeah following what we do in other components should be adequate